Duels of Dreams update

danimagaming opened the pull request:
<!-- ATTENTION: This Pull Request template changed on 03/17/22. Please ensure that you are completing this template to the fullest extent possible. -->

# Submission Checklist
## Required

<!-- Check these off by adding an 'x' to each of these boxes. If you fail to meet all these criteria, your PR will be rejected. -->

- [ x] The pull request title clearly contains the name of the sheet I am editing.
- [ x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x ] The pull request makes changes to files in only one sub-folder.
- [x ] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

## New Sheet Details

<!-- If you are submitting a new sheet to the repository, please fill in any empty spaces indicated by < >. -->

- The name of this game is: <   >  _(i.e. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: <   > _(i.e. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: <   > _(i.e. Dungeons & Dragons, FATE)_

- [ ] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.

<!-- Please check any that apply: -->

- [ x] This sheet has been made on behalf of, or by, the game's publisher.
- [x ] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: <  https://duelsofdreams.com/ >
- [ ] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system.

# Changes / Description
added skill images, a points tab, a collapsing compact skill view, and a buncha auto-calculations for skills and the point buy table. Also fixed a bug where the tabs weren't working.
<!-- This is an optional step, but detailing the nature of the changes makes it easier for other contributors to track down bugs and fix issues -->

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/Duels%20of%20Dreams%20-%20DODAS/roll20-character-sheets) *Roll20 Internal Use only.*

danimagaming commented:
added skill images, a points tab, a collapsing compact skill view, and a buncha auto-calculations for skills and the point buy table. Also fixed a bug where tabs weren't working.

BronsonHall commented:
Hey @danimagaming ,

Looks like you updated the sheet to CSE with this PR, removing .sheet- prefix, etc.  You'll want to update the sheet.json as well, namely changing `"legacy": true` to `"legacy": false`, or the sheet's not going to take your CSS once live.  Once you've swapped that out, this should be good to go.  

That said, it looks like you've also got two screenshots in this folder, one of the older design and one newer.  You'll want to change `"preview"` in your sheet.json too, to point towards this newer screenshot, or rename the new screenshot to `DODAS.png`.  That way, the preview image in the game listing will accurately portray the current appearance of the sheet.

Thanks!

danimagaming commented:
Ah, good catch. Thanks, I'll get on that!

On Wed, Nov 15, 2023 at 5:40 PM Lucas Hall ***@***.***> wrote:

> Hey @danimagaming <https://github.com/danimagaming> ,
>
> Looks like you updated the sheet to CSE with this PR, removing .sheet-
> prefix, etc. You'll want to update the sheet.json as well, namely changing "legacy":
> true to "legacy": false, or the sheet's not going to take your CSS once
> live. Once you've swapped that out, this should be good to go.
>
> That said, it looks like you've also got two screenshots in this folder,
> one of the older design and one newer. You'll want to change "preview" in
> your sheet.json too, to point towards this newer screenshot, or rename the
> new screenshot to DODAS.png. That way, the preview image in the game
> listing will accurately portray the current appearance of the sheet.
>
> Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Roll20/roll20-character-sheets/pull/12299#issuecomment-1813380230>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APDO2QRM5JFFNXEUCFN5Q4TYEVAFVAVCNFSM6AAAAAA7LLN4WOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMJTGM4DAMRTGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
